src/main/webapp/ts/matches.ts

```diff
@@ -120,12 +120,20 @@ function displayNewMatchPopup(matches: Array<User>) {
 async function queryServer(matchRequest: MatchRequest) {
     const json = JSON.stringify(matchRequest);
     return fetch('/new-matches-query', { method: 'POST', body: json })
+  
         .then((response) => {
             return response.json();
         })
         .then((users) => {
             //convert range from json to User
             const out: Array<User> = [];
+            console.log("here1");
```

Comment:
Here is the documented way to follow a redirect after a fetch call: https://stackoverflow.com/questions/39735496/redirect-after-a-fetch-post-call

Reply:
So, add redirect: "follow", wouldn't that just redirect to the "/new-matches-query"?

Can i still use the document.location.href for redirects?

Reply:
Implemented